plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2659,11 +2657,8 @@ void producer_plugin::log_failed_transaction(const transaction_id_type&    trx_i
 
 // Called from only one read_only thread
 void producer_plugin_impl::switch_to_write_window() {
-   if (_log.is_enabled(fc::log_level::debug)) {
-      auto now = fc::time_point::now();
-      fc_dlog(_log, "Read-only threads ${n}, read window ${r}us, total all threads ${t}us",
-              ("n", _ro_thread_pool_size)("r", now - _ro_read_window_start_time)("t", _ro_all_threads_exec_time_us.load()));
-   }
+   fc_ilog(_log, "Read-only threads ${n}, read window ${r}us, total all threads ${t}us",
+           ("n", _ro_thread_pool_size)("r", fc::time_point::now() - _ro_read_window_start_time)("t", _ro_all_threads_exec_time_us.load()));
```

Comment:
This is debatable change. Happy to make this `debug` if it seems too chatty.

Reply:
Yes I think it would be too chatty. API operators probably are not interested in the details of window switching.

Reply:
Changed to debug level.